libs/designsystem/src/lib/scss/print/components/_ion-item.scss

```diff
@@ -0,0 +1,20 @@
+@media print {
+  ion-item-group {
+    margin-block: 17pt !important;
```

Comment:
17pt feels a bit like a magic number when reading through the code. Consider adding a comment or similar.